tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -2060,6 +2060,46 @@ public function userCopiesOrMovesFileWithFileIdFromAndToSpaceBetweenSpaces(strin
 		}
 	}
 
+	/**
+	 * @When /^user "([^"]*)" tries to move a (?:file|folder) "([^"]*)" to (space|folder) "([^"]*)" using its id in destination path "([^"]*)"$/
+	 * @When /^user "([^"]*)" moves a (?:file|folder) "([^"]*)" to (folder) "([^"]*)" using its id in destination path "([^"]*)"$/
+	 *
+	 * @param string $user
+	 * @param string $source
+	 * @param string $destinationType
+	 * @param string $destinationName
+	 * @param string $destinationPath
+	 *
+	 * @throws GuzzleException
+	 * @return void
+	 */
+	public function userMovesFileToResourceUsingItsIdAsDestinationPath(
+		string $user,
+		string $source,
+		string $destinationType,
+		string $destinationName,
+		string $destinationPath
+	): void {
+		$source = \trim($source, "/");
+		$baseUrl = $this->featureContext->getBaseUrl();
+		$suffix = "";
+		if ($this->featureContext->getDavPathVersion() === WebDavHelper::DAV_VERSION_SPACES) {
+			$suffix = $this->getSpaceIdByName($user, "Personal") . "/";
```

Comment:
just for personal space at the moment? I mean i should also work same for spaces right? or test will be covered for other spaces as well?

Reply:
:+1: Let's look at it.